metadata_layer.py

```diff
@@ -689,10 +689,6 @@ def license_pool(self, _db):
             identifier=identifier_obj
         )
         if not license_pool:
-            rights_status = get_one(
-                _db, RightsStatus, uri=self.default_rights_uri
-            )
```

Comment:
This wasn't being used. I think it was from before we moved things to CirculationData.